CefSharp.Wpf/ChromiumWebBrowser.cs

```diff
@@ -1243,8 +1249,9 @@ private void OnMouseButton(MouseButtonEventArgs e)
 
         public void Load(string url)
         {
-            //Added null check -> binding-triggered changes of Address will lead to a nullref after Dispose has been called.
-            if (managedCefBrowserAdapter != null)
+            // Added null check -> binding-triggered changes of Address will lead to a nullref after Dispose has been called
+            // or before OnApplyTemplate has been called
+            if (managedCefBrowserAdapter != null && GetMainFrame() != null)
```

Comment:
Better to get a reference to the frame, check if it's null, and then use that reference.
The `managedCefBrowserAdapter != null` check becomes redundant as `GetMainFrame` will return null if `managedCefBrowserAdapter == null`.